Flow.Launcher/PriorityChangeWindow.xaml

```diff
@@ -0,0 +1,43 @@
+﻿<Window x:Class="Flow.Launcher.PriorityChangeWindow"
+        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+        xmlns:local="clr-namespace:Flow.Launcher"
+        Loaded="PriorityChangeWindow_Loaded"
+        mc:Ignorable="d"
+        Title="PriorityChangeWindow" Height="250" Width="300">
+    <Grid>
+        <Grid.RowDefinitions>
+            <RowDefinition />
+            <RowDefinition Height="60"/>
+            <RowDefinition Height="60"/>
+            <RowDefinition />
+        </Grid.RowDefinitions>
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition Width="20" />
+            <ColumnDefinition />
+        </Grid.ColumnDefinitions>
+        <TextBlock FontSize="14" Grid.Row="0" Grid.Column="1" VerticalAlignment="Center"
+                   HorizontalAlignment="Left" Text="Current Priority:" />
+        <TextBlock x:Name="OldPriority" Grid.Row="0" Grid.Column="1" Margin="170 10 10 10" FontSize="14"
+                   VerticalAlignment="Center" HorizontalAlignment="Left" />
+
+        <TextBlock FontSize="14" Grid.Row="1" Grid.Column="1" VerticalAlignment="Center"
+                   HorizontalAlignment="Left" Text="New Priority:" />
+        <StackPanel Grid.Row="1" Orientation="Horizontal" Grid.Column="1">
+            <TextBox x:Name="tbAction" Margin="140 10 15 10" Width="105" VerticalAlignment="Center" HorizontalAlignment="Left" />
+        </StackPanel>
+
+        <TextBlock Grid.Row="2" Grid.ColumnSpan="1" Grid.Column="1" Foreground="Gray"
+                   Text="Higher means display more ahead, if you want the results show lower than any other plugins, provide a negative number" TextWrapping="Wrap"/>
```

Comment:
1. Could we update the message to:
'Higher the number, the closer the result will be to the top. If you want the results to be lower than any other plugin's, provide a negative number'

2. Also lets add this to the language file so others can translate

3. Lets fix up the width so it doesnt look crowded:

![image](https://user-images.githubusercontent.com/26427004/103753790-94977080-505f-11eb-80b6-669f46f864d0.png)